Orange/widgets/gui.py

```diff
@@ -2239,7 +2239,7 @@ def paintEvent(self, ev):
 
 
 def auto_commit(widget, master, value, label, auto_label=None, box=True,
-                checkbox_label=None, orientation=None, **misc):
+                checkbox_label=None, orientation=None, callback='commit', **misc):
```

Comment:
I would prefer `callback=None` as the default argument. Furthermore, I don't like passing strings like this; we have to do it for attributes, while for methods we can pass the reference, e.g. `callback=self.apply`. The method's name is then retrieved through `callback.__name__`.